module/Finna/src/Finna/RecordDataFormatter/Specs/DefaultRecord.php

```diff
@@ -125,6 +125,30 @@ protected function getDefaultCoreFields()
                 $options['pos'] = $pos;
                 $lines[$key] = [true, $dataMethod, $callback, $options];
             };
+        $setTemplateLine(
+            'hierarchical_content',
+            'getHierarchicalEmbeddedComponentParts',
+            'data-componentparts.phtml',
+            [
+                'context' => ['class' => 'recordIncludedIn'],
+                'labelFunction' => function ($data, $driver) {
+                    return !empty($driver->getHierarchyParentID())
+                    ? 'Included in the collections'
```

Comment:
Eihän tämä näin voi olla? component partsit ovat aina ko. tietueen lapsia, jolloin tämä on väärin päin, eikö? Ja useamman tason hierarkiassa voi olla molempia. Tarvitaan siis kaksi erillistä kenttää.